common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -796,4 +801,22 @@ public boolean verify(X509Certificate[] certificates, String hostname, SSLSessio
             }
         };
     }
+
+    /**
+     * Returns true if no-copy optimizations for direct {@code ByteBuffers} in {@code Ciphers}
+     * are currently enabled.
+     */
+    @ExperimentalApi
```

Comment:
let's file an issue to remove this experimentalApi and assign sensible milestone